apis/apps/v1alpha1/nebulacluster_metad.go

```diff
@@ -262,6 +274,27 @@ func (c *metadComponent) GenerateVolumes() []corev1.Volume {
 			},
 		},
 	}
+
+	if c.nc.Spec.Metad.Config["license_path"] != "" {
+		volumes = append(volumes, corev1.Volume{
+			Name: "nebula-license",
+			VolumeSource: corev1.VolumeSource{
+				ConfigMap: &corev1.ConfigMapVolumeSource{
+					LocalObjectReference: corev1.LocalObjectReference{
+						Name: "nebula-license",
```

Comment:
I think this should be user specified, what do you think?
`Key` as well.

Reply:
Right, I will update these to crd